src/react-components/ory/user-auth-card.tsx

```diff
@@ -94,6 +95,7 @@ export const UserAuthCard = ({
 
   let $flow = null
   let $oidc = null
+  let $code = null
```

Comment:
I think this is implicitly typed as any (or otherwise it would be typed as null, which should cause type issues), which should be avoided IMO. Please add a type manually, if possible. And if you're feeling especially boyscouty, you can add types for the other values as well 😁

Reply:
These are only react elements. Why is it necessary to type them?

Reply:
Because everything should always be typed. No exceptions. Allowing something to (implicitly) be typed as `any` defeats the entire purpose of using TypeScript in the first place. Plus, it looks like a really simple type annotation in this case, so I don't see why not.